packages/orbit-components/src/Stepper/Stepper.stories.tsx

```diff
@@ -23,10 +23,17 @@ export const Default: Story = {
       disable: true,
     },
   },
+
+  args: {
+    ariaLabel: "Passengers number",
```

Comment:
Maybe "Number of passengers" would be better name 🤔 (Same for the example in the docs)